code/front-end/Satellite GUI.md

```diff
@@ -0,0 +1,35 @@
+# Satellite GUI
+
+Satellite GUI is a [vue.js](https://vuejs.org/) + [TypeScript](https://www.typescriptlang.org/) front-end codebase that provides an UI interface for users to interact with the network.
+
+## Development Workflow
+
+There are two ways to start local development environment depending on what you need.
+
+### Prerequisites
+
+- Clone the [storj repo](https://github.com/storj/storj)
+- Make sure you have both [node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/get-npm) installed
+- Go to `web/satellite` folder
+- run `npm install`. It will install all dependencies for the codebase
+
+### Running without a back-end server
+
+This is a good way to run it if you want to do UI development since it can do hot-reloading for the changes you make. However, you won't be able to get any necessary data from the back-end.
+
+- Run `npm run serve` from `web/satellite` to start a development environment
+- Visit `localhost:8080`
+
+### Running with a back-end server
+
+This way you will be able to test out how the back-end interacts with the front-end
+
+- Make sure you have [storj-sim](https://github.com/storj/storj/wiki/Test-network) set up correctly
+- If you want your local change to be reflected, run `npm run build` from `web/satellite`
+- Run `storj-sim network setup` to set up the test network
+- Run `storj-sim network run` to start the test network
+- Visit `localhost:10002` to access the satellite GUI
```

Comment:
is the GUI always at this port 10002? How can someone find what the port is set to if it isn't here?

Reply:
True, I will add the instruction on how to find the correct port